contracts/governance/extensions/GovernorVotesERC721.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts v4.3.2 (governance/extensions/ERC721GovernorVotesERC721.sol)
+
+pragma solidity ^0.8.0;
+
+import "../Governor.sol";
+import "../../token/ERC721/extensions/draft-ERC721Votes.sol";
+import "../../utils/math/Math.sol";
+
+/**
+ * @dev Extension of {Governor} for voting weight extraction from an {ERC721Votes} token.
+ *
+ * _Available since v4.5._
+ */
+abstract contract GovernorVotesERC721 is Governor {
```

Comment:
AFAIK, this module produces the exact same result as GovernorVotesERC20, because the `getPastVotes` method is shared. Maybe we should not have 2 identical modules

Reply:
Wouldnt it serve as a guide for users trying to implment Governance with ERCVotes? Or should we include some documentation somewhere instead?